src/Content/ImmutableMemoryContentStorageHandler.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Kirby\Content;
+
+use Kirby\Cms\Language;
+use Kirby\Exception\LogicException;
+
+/**
+ * @package   Kirby Content
+ * @author    Nico Hoffmann <nico@getkirby.com>
```

Comment:
```suggestion
 * @author    Bastian Allgeier <bastian@getkirby.com>
```

`Nitpick:` Technically not quite right 😄

Reply:
Since you created the first version on Notion and I hardly had to touch it, I put you in there :)